src/platform/site-wide/cta-widget/index.js

```diff
@@ -118,10 +117,42 @@ export class CallToActionWidget extends React.Component {
             give you access to your personal health information.
           </p>
         ),
-        primaryButtonText: 'Verify Your Identity',
+        primaryButtonText: 'Verify your identity',
         primaryButtonHandler: verify,
         status: 'continue',
       };
+    } else if (this.props.appId === 'direct-deposit') {
```

Comment:
two things:
- this doesn't need to be in an `else if`, just an `if`. that will be more consistent with the rest of the other conditions
- can you use the constant here instead of a string?

Reply:
It might be nice to wrap the content of this conditional in a local helper similar to `getHealthToolContent()`

Reply:
Code refactored.